### PR TITLE
Ui tweaks

### PR DESCRIPTION
**Pull request by smartcatdev**

### CODE REVIEW

**Comment by smartcatdev** [comment]
@smartcat-ericgreen please review this PR

File: assets/css/style.css [hunk 1]
```diff
@@ -853,6 +895,14 @@ button.close-ticket:hover {
 }
 
 /* #### Comments ***********************************************************************/
+
+.discussion-area {
+}
+
+.tab-content>.tab-pane {
+    min-height: 600px;
```
Comment (smartcat-ericgreen) [inline assets/css/style.css]: This causes the editor area to extend way past text field, the tickets tab-pane needs a unique class 

image

File: assets/css/style.css [hunk 2]
```diff
@@ -465,6 +500,13 @@ textarea {
     font-size: 12px;
 }
 
+#statistics-container .stat-tab:first-of-type {
+    padding-left: 0;
```
Comment (smartcat-ericgreen) [inline assets/css/style.css]: on mobile this causes the top and bottom stats boxes to appear off-center 

image

Comment (smartcatdev) [inline assets/css/style.css]: moved within `@media( min-width: 768px )`

File: assets/css/style.css (hunk 1 above)
Comment (smartcatdev) [inline assets/css/style.css]: fixed by applying same rule on `.ticket-area-wrapper`

**Pull request merged**